### PR TITLE
chore(watson): Add new watson-x tiles and remove watson resources

### PR DESCRIPTION
**Pull request by dgutride**
## Description

Add new documentation tile, application tile and a new 45 minute how-to tile as shown below.
Screenshot 2024-04-03 at 1 11 43 PM
Screenshot 2024-04-03 at 1 12 03 PM

Can be seen on this cluster with google creds:  https://rhods-dashboard-redhat-ods-applications.apps.rosa.dg-cluster.937s.p3.openshiftapps.com/resources?provider=%5B%22IBM+watsonx.ai%22%5D

Closes: https://issues.redhat.com/browse/RHOAIENG-4447

## How Has This Been Tested?

Applied yaml to cluster to ensure watson-x tiles (application and how-to) show up correctly and links work as expected.

## Test Impact

Need to make sure new watson-x tiles are created and no old watson tiles are created in fresh install.  Platform team is caring for removing them on upgrade, though.

## Request review criteria:

Self checklist (all need to be checked):
- [ ] The developer has manually tested the changes and verified that the changes work
- [ ] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Review by lucferbux — APPROVED** [review]
/lgtm

Resources look great!
Screenshot 2024-04-04 at 16 39 28


@zdtsw @VaishnaviHire FYI we are gonna merge this, i think the watson folder might need a deletion from your side, is that right?

**Comment by zdtsw** [comment]
> /lgtm
> 
> Resources look great! Screenshot 2024-04-04 at 16 39 28
> 
> @zdtsw @VaishnaviHire FYI we are gonna merge this, i think the watson folder might need a deletion from your side, is that right?

yes, some new one to add and the old one need to be cleanuped,
we have @ykaliuta assigned for this task already

File: manifests/overlays/apps/base/watson-x/watson-x-docs.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: dashboard.opendatahub.io/v1
+kind: OdhDocument
+metadata:
+  name: watson-x-use-case
+  annotations:
+    opendatahub.io/categories: 'Data preprocessing, Data visualization, Model development'
+spec:
+  appName: watson-x-ai
+  type: how-to
+  displayName: Enable your enterprise to develop and deploy machine learning models and generative AI solutions.
```
Comment (MelissaFlinn) [inline manifests/overlays/apps/base/watson-x/watson-x-docs.yaml:10]: change displayName to "Watsonx.ai use case"
description is correct

Comment (dgutride) [reply manifests/overlays/apps/base/watson-x/watson-x-docs.yaml]: ooph - wow.  Good catch...
Screenshot 2024-04-04 at 1 05 28 PM

Comment (MelissaFlinn) [reply manifests/overlays/apps/base/watson-x/watson-x-docs.yaml]: I notice now with the screen capture showing the two tiles side-by-side - they should be consistent
Please change the title on the How to tile from **Watsonx.ai use case** to *IBM watsonx.ai use case*

Comment (dgutride) [reply manifests/overlays/apps/base/watson-x/watson-x-docs.yaml]: Screenshot 2024-04-04 at 1 19 11 PM
This should be resolved now...

**Pull request merged**